pages/docs/reference/whatsnew14.md

```diff
@@ -500,3 +500,20 @@ Complete the initial configuration, and when you add a new dependency to `cocoap
 The new dependency will be added automatically. No additional steps are required.
 
 Learn [how to add dependencies](native/cocoapods.html).
+
+## Kotlin/JS
+
+### New Gradle DSL
+The `kotlin.js` Gradle plugin comes with an adjusted Gradle DSL, which provides a number of new configuration options and is more closely aligned to the DSL used by Kotlin's `multiplatform` plugin. Some of the most impactful changes include:
+
+- Explicit toggles for the creation of executable files via `binaries.executable()`. Read more about the executing Kotlin/JS and its environment [here](js-project-setup.html#choosing-execution-environment).
```

Comment:
@SebastianAigner just in case, remind you that it will be optional in 1.4.0 for current(aka legacy) compiler. And if the feature is not enabled using a key in `gradle.properties` writing `binaries.executable()` will not have any effect on the current compiler. But it would be nice to promote it since it could improve build performance.

Reply:
Let's keep it here as-is for "promotion" reasons – I'll add a small clarification to the docs.